tests/scripts/setupTests.js

```diff
@@ -1,2 +1,6 @@
-// ONLY FOR TESTING INTERNALLY IN --TEST_MODE
-console.log("Running 'setupFilesAfterEnv'");
```

Comment:
Removed this log as was causing noise in tests. Didn't seem required, is it?

Reply:
This is actually a hack I have to test if the serverless-bundle is using the config set in the package.json:

```
  "jest": {
    "setupFilesAfterEnv": [
      "<rootDir>/tests/scripts/setupTests.js"
    ]
  },
```

And we want to test the `./bin/scripts.js`:

```
    "test-scripts": "./bin/scripts.js test --TEST_MODE tests/scripts/test.js"
```

So while it's in the same directory as the Jest tests, it's really a way for me to run it manually to ensure that the above script works. I think I'll move it out of the `tests/` directory. That way we won't need to ignore it:

```
    "test": "jest ./tests --no-watchman --testPathIgnorePatterns=\"./tests/scripts\"",
```

I'll have a look at this before merging.

Reply:
Awesome